modules/compbox/templates/compbox-index.html.erb

```diff
@@ -28,8 +28,7 @@
     <div class="for-machines">
         <ul>
             <li><a href="/comp-api/">Comp API</a></li>
-            <li><a href="/stream">Comp API Stream (legacy)</a></li>
-            <li><a href="/pystream/">Comp API Stream (experimental)</a></li>
+            <li><a href="/stream/">Comp API Stream</a></li>
```

Comment:
Should this be `/stream`?
```suggestion
            <li><a href="/stream">Comp API Stream</a></li>
```

Reply:
We should probably decide on what the canonical URL should be. Personally I prefer URLs with trailing slashes. Whatever we choose, we should be consistent.